src/main.tsx

```diff
@@ -20,27 +20,22 @@ import PlaygroundView from "./features/playground/playgroundView";
 import ConfigTableExampleView from "./features/configTable/configTableExample";
 import MovieTableView from "./features/paginatedTable/moviePage";
 import VideoGamesTableView from "./features/formExample/videoGamesPage";
+import { mainSections } from "./siteMap/mainSections";
+import { useEffect } from "react";
 
 const Component = () => {
   return (
     <>
       <Routes>
         <Route index path="/" element={<Navigate to="/test" />} />
         <Route path="/Unauthorized" element={<Unauthorised />} />
-        <Route path="/jsonplaceholder" element={<JsonPlaceHolderView />} />
-        <Route path="/playground" element={<PlaygroundView />} />
-        <Route path="/configTable" element={<ConfigTableExampleView />} />
-        <Route path="/moviesTable" element={<MovieTableView />} />
-        <Route path="/videoGamesTable" element={<VideoGamesTableView />} />
-
-        {
-          //Understand why with this method :params in path are not being recognized.
-          /*mainSections.map((x) =>
-          x.subsections.map((s) => (
-            <Route path={s.path} element={s.component({ a: dispatch })} />
-          ))
-        )*/
-        }
+        {mainSections.map((x) =>
+          x.subsections.map((s) =>
```

Comment:
and what if 's' has further nestes subsections?

Reply:
would you support infinite with recursion in the project template? 
I was thinking that 2 were enough and more would be a project dedicated task if needed